.github/workflows/ci.yml

```diff
@@ -1,12 +1,12 @@
-name: Verify PRs
+name: Continuous Integration
 
-on: [pull_request]
+on: [push]
```

Comment:
Does this run on any push? To any remote branch? Or a push to master?

Reply:
Any push.